src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue

```diff
@@ -78,11 +76,22 @@ export default defineComponent({
       userStore.setUserProfile(userProfile);
     }
 
-    certificationTypeStore.$reset();
-
-    await wizardStore.initializeWizard(applicationWizard, applicationStore.draftApplication);
+    if (applicationStore.draftApplication.certificationTypes?.includes(CertificationType.FIVE_YEAR)) {
```

Comment:
Also looks like you're doing this application includes a lot, you could do a getter for this as well might make things cleaner...

Reply:
I put the function into the application store.